### ISSUE TITLE
Build the first interfaces of the API

### ISSUE THREAD

**Issue by amihaiemil** [amihaiemil/docker-java-api#1]
As per the [documentation](https://docs.docker.com/engine/api/v1.33/), the entry point is ``Docker``, which has ``Containers``, ``Images``, ``Volumes`` etc.

```java
public interface Docker {
    Containers containers();
    Images images();
    Volumes volumes();
    //...
}
```

**Comment by 0pdd** [comment]
@amihaiemil 3 puzzles [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72) are still not solved; solved: [#13](https://github.com/amihaiemil/docker-java-api/issues/13), [#14](https://github.com/amihaiemil/docker-java-api/issues/14), [#18](https://github.com/amihaiemil/docker-java-api/issues/18), [#19](https://github.com/amihaiemil/docker-java-api/issues/19), [#3](https://github.com/amihaiemil/docker-java-api/issues/3), [#4](https://github.com/amihaiemil/docker-java-api/issues/4), [#7](https://github.com/amihaiemil/docker-java-api/issues/7), [#9](https://github.com/amihaiemil/docker-java-api/issues/9).

**Comment by 0pdd** [comment]
@amihaiemil 2 puzzles [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71) are still not solved; solved: [#13](https://github.com/amihaiemil/docker-java-api/issues/13), [#14](https://github.com/amihaiemil/docker-java-api/issues/14), [#18](https://github.com/amihaiemil/docker-java-api/issues/18), [#19](https://github.com/amihaiemil/docker-java-api/issues/19), [#3](https://github.com/amihaiemil/docker-java-api/issues/3), [#4](https://github.com/amihaiemil/docker-java-api/issues/4), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#7](https://github.com/amihaiemil/docker-java-api/issues/7), [#9](https://github.com/amihaiemil/docker-java-api/issues/9).

**Comment by 0pdd** [comment]
@amihaiemil 5 puzzles [#77](https://github.com/amihaiemil/docker-java-api/issues/77), [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#86](https://github.com/amihaiemil/docker-java-api/issues/86) are still not solved; solved: [#13](https://github.com/amihaiemil/docker-java-api/issues/13), [#14](https://github.com/amihaiemil/docker-java-api/issues/14), [#18](https://github.com/amihaiemil/docker-java-api/issues/18), [#19](https://github.com/amihaiemil/docker-java-api/issues/19), [#3](https://github.com/amihaiemil/docker-java-api/issues/3), [#4](https://github.com/amihaiemil/docker-java-api/issues/4), [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#7](https://github.com/amihaiemil/docker-java-api/issues/7), [#9](https://github.com/amihaiemil/docker-java-api/issues/9).

**Comment by 0pdd** [comment]
@amihaiemil 5 puzzles [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#86](https://github.com/amihaiemil/docker-java-api/issues/86), [#95](https://github.com/amihaiemil/docker-java-api/issues/95) are still not solved; solved: [#13](https://github.com/amihaiemil/docker-java-api/issues/13), [#14](https://github.com/amihaiemil/docker-java-api/issues/14), [#18](https://github.com/amihaiemil/docker-java-api/issues/18), [#19](https://github.com/amihaiemil/docker-java-api/issues/19), [#3](https://github.com/amihaiemil/docker-java-api/issues/3), [#4](https://github.com/amihaiemil/docker-java-api/issues/4), [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#77](https://github.com/amihaiemil/docker-java-api/issues/77), [#7](https://github.com/amihaiemil/docker-java-api/issues/7), [#9](https://github.com/amihaiemil/docker-java-api/issues/9).

**Comment by 0pdd** [comment]
@amihaiemil 5 puzzles [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#95](https://github.com/amihaiemil/docker-java-api/issues/95), [#96](https://github.com/amihaiemil/docker-java-api/issues/96) are still not solved; solved: [#13](https://github.com/amihaiemil/docker-java-api/issues/13), [#14](https://github.com/amihaiemil/docker-java-api/issues/14), [#18](https://github.com/amihaiemil/docker-java-api/issues/18), [#19](https://github.com/amihaiemil/docker-java-api/issues/19), [#3](https://github.com/amihaiemil/docker-java-api/issues/3), [#4](https://github.com/amihaiemil/docker-java-api/issues/4), [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#77](https://github.com/amihaiemil/docker-java-api/issues/77), [#7](https://github.com/amihaiemil/docker-java-api/issues/7), [#86](https://github.com/amihaiemil/docker-java-api/issues/86), [#9](https://github.com/amihaiemil/docker-java-api/issues/9).

**Comment by 0pdd** [comment]
@amihaiemil 6 puzzles [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#95](https://github.com/amihaiemil/docker-java-api/issues/95), [#96](https://github.com/amihaiemil/docker-java-api/issues/96), [#98](https://github.com/amihaiemil/docker-java-api/issues/98), [#99](https://github.com/amihaiemil/docker-java-api/issues/99) are still not solved; solved: [#13](https://github.com/amihaiemil/docker-java-api/issues/13), [#14](https://github.com/amihaiemil/docker-java-api/issues/14), [#18](https://github.com/amihaiemil/docker-java-api/issues/18), [#19](https://github.com/amihaiemil/docker-java-api/issues/19), [#3](https://github.com/amihaiemil/docker-java-api/issues/3), [#4](https://github.com/amihaiemil/docker-java-api/issues/4), [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#77](https://github.com/amihaiemil/docker-java-api/issues/77), [#7](https://github.com/amihaiemil/docker-java-api/issues/7), [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#86](https://github.com/amihaiemil/docker-java-api/issues/86), [#9](https://github.com/amihaiemil/docker-java-api/issues/9).

**Comment by 0pdd** [comment]
@amihaiemil 6 puzzles [#103](https://github.com/amihaiemil/docker-java-api/issues/103), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#95](https://github.com/amihaiemil/docker-java-api/issues/95), [#98](https://github.com/amihaiemil/docker-java-api/issues/98), [#99](https://github.com/amihaiemil/docker-java-api/issues/99) are still not solved; solved: [#13](https://github.com/amihaiemil/docker-java-api/issues/13), [#14](https://github.com/amihaiemil/docker-java-api/issues/14), [#18](https://github.com/amihaiemil/docker-java-api/issues/18), [#19](https://github.com/amihaiemil/docker-java-api/issues/19), [#3](https://github.com/amihaiemil/docker-java-api/issues/3), [#4](https://github.com/amihaiemil/docker-java-api/issues/4), [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#77](https://github.com/amihaiemil/docker-java-api/issues/77), [#7](https://github.com/amihaiemil/docker-java-api/issues/7), [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#86](https://github.com/amihaiemil/docker-java-api/issues/86), [#96](https://github.com/amihaiemil/docker-java-api/issues/96), [#9](https://github.com/amihaiemil/docker-java-api/issues/9).

**Comment by 0crat** [comment]
Job `gh:amihaiemil/docker-java-api#1` is not assigned, can't get performer
